src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -79,6 +79,7 @@
       <_NET70RuntimePackVersion>7.0.$(VersionFeature70)</_NET70RuntimePackVersion>
       <_NET70TargetingPackVersion>7.0.$(VersionFeature70)</_NET70TargetingPackVersion>
       <_NET70ILLinkPackVersion>7.0.100-1.23062.2</_NET70ILLinkPackVersion>
+      <_NET70WebAssemblyPackVersion>$(MicrosoftNETCoreAppRuntimePackageVersion)</_NET70WebAssemblyPackVersion>
```

Comment:
You want the 70 webassembly pack version to be the current net8 version number?  Also, is 60 meant to be the same as 70?

Reply:
Given that there is currently only one version of this package and as far as I know it handles the downlevels correctly I think for the moment we should just make sure there is just a definition that will match for the supported targets.